charts/pega/README.md

```diff
@@ -642,14 +646,19 @@ installer:
   adminPassword: "ADMIN_PASSWORD"
 ```
 
-### Upgrade
+### Upgrades and patches
+
+To **upgrade Pega Platform software** deployed in a Kubernetes environment, you must download the latest Pega software from  [Pega Digital Software Delivery](https://community.pega.com/digital-delivery) and then use the appropriate upgrade guide:
+
+- Use the appropriate Pega Platform deployment Upgrade Guide available from the page, [Stay current with Pega](https://community.pega.com/upgrade).
+- Use the latest Pega application software Upgrade Guide, which is separate from Pega Platform software. You can locate the appropriate upgrade guide for your installed application from the page, [All Products](https://community.pega.com/knowledgebase/products).
 
-For upgrades of the Pega platform, you must specify the installer Docker image and the type of upgrade to execute.
+To **apply a Pega Platform patch** with zero downtime to your existing Pega platform software, you must use `installer.upgrade.upgradeType "out-of-place"` option. For step-by-step guidance to apply a Pega Platform patch, see the Pega-provided runbook, [Patching Pega Platform in your deployment](/docs/patching-pega-deployment.md)
 
 Upgrade type    | Description
 ---             | ---
-`in-place`      | An in-place upgrade will upgrade both rules and data in a single run.  This will upgrade your environment as quickly as possible but will result in downtime.
-`out-of-place`  | An out-of-place upgrade involves more steps to minimize downtime.  It will place the rules into a read-only state, then migrate the rules to a new schema. Next it will upgrade the rules to the new version. Lastly it will separately upgrade the data.
+`in-place`      | An in-place upgrade will upgrade both rules and data in a single run.  An in-place upgrade process will upgrade your environment as quickly as possible but pauses the deployment while and the process upgrades all of the rules of your Pega application.
+`out-of-place`  | An out-of-place upgrade uses a split-schema process to apply patches.  It places the rules into a read-only state, creates appropriate, temporary schemas in your current database, then migrates the rules to a temporary rules schema it creates when you specify `installer.targetRulesSchema: "rules_upgrade"`. The patch process applies only changes observed between the patch and your currently running version and then separately upgrades the data. To apply a patch, specify `installer.upgradeType: out-of-place"` and `installer.targetRulesSchema: "rules_upgrade"`.
 
 Example:
```

Comment:
What is 'read-only' state infer here?. During ZDT patch upgrade we can create the rules in source rules schema during the upgrade process

Reply:
Updated the details to shorten it...see what you think.